recipes/glimpse-bio/build.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
 
+export __COMMIT_ID__="${GIT_FULL_HASH}"
+export __COMMIT_DATE__="$(date -Idate -u)"
```

Comment:
I couldn't figure out how to get this without cloning the repo, so I just set it as the bioconda _build_ date